scripts/env/dev/init.sh

```diff
@@ -170,7 +170,7 @@ mkdir -p /etc/ssl
 cp -R  ${BASEDIR}/../../../ansible/files/certs/* /etc/ssl/
 
 # Generate random password for jks key and keystore passwords
-sh ${S3_SRC_DIR}/scripts/create_auth_jks_password.sh
+#sh ${S3_SRC_DIR}/scripts/create_auth_jks_password.sh
```

Comment:
simillar changes are required in release init.sh

Reply:
fixed.